test/libsolidity/SemanticTest.cpp

```diff
@@ -453,6 +453,9 @@ TestCase::TestResult SemanticTest::runTest(
 			test.setFailure(!m_transactionSuccessful);
 			test.setRawBytes(move(output));
 			test.setContractABI(m_compiler.contractABI(m_compiler.lastContractName(m_sources.mainSourceFile)));
+
+			if (!test.validFractionDigits())
```

Comment:
Didn't you want to remove this?

Reply:
Thats true!

Reply:
Ok, I also removed that `valid()` function now.